### ISSUE TITLE
is there any eta for xgboost4j support of scala 2.13 or scala 3? any update would be highly appreciated.

### ISSUE THREAD

**Issue by MohammadForouhesh** [dmlc/xgboost#8650]
Thanks for participating in the XGBoost community! We use https://discuss.xgboost.ai for any general usage questions and discussions. The issue tracker is used for actionable items such as feature proposals discussion, roadmaps, and bug tracking.  You are always welcomed to post on the forum first :)

Issues that are inactive for a period of time may get closed. We adopt this policy so that we won't lose track of actionable issues that may fall at the bottom of the pile. Feel free to reopen a new one if you feel there is an additional problem that needs attention when an old one gets closed.

For bug reports, to help the developer act on the issues, please include a description of your environment, preferably a minimum script to reproduce the problem.

For feature proposals, list clear, small actionable items so we can track the progress of the change.

**Comment by wbo4958** [comment]
Yeah, I saw many users are requesting this feature, I will check that.

**Comment by MohammadForouhesh** [comment]
Thanks.
If I could be of any help, don't hesitate to reach out.

**Comment by wbo4958** [comment]
@MohammadForouhesh Wow, if you can help, that will be great. Could you contribute this feature?

**Comment by MohammadForouhesh** [comment]
@wbo4958 yeah i am open to help, but it would be best if one of the more seasoned contributers guids me through the source code first.

**Comment by dotbg** [comment]
There are few bits we need before working on the scala 2.13 support :
1. Flink 1.8 -> 1.17:  https://github.com/dmlc/xgboost/pull/9046
2. Scalatest  3.0.8 ->  3.2.15 : https://github.com/dmlc/xgboost/pull/8925
3. Spark to 3.3.0 + 

I believe, https://github.com/dmlc/xgboost/pull/8310 can be reused
